lms/static/sass/views/_homepage.scss

```diff
@@ -16,7 +16,7 @@ $learn-more-horizontal-position: calc(50% - 100px); // calculate the left positi
       @extend %ui-no-list;
 
       .courses-listing-item {
-        margin: ($baseline*0.75) 0 ($baseline*1.5) 0;
+        margin: 0 0 ($baseline*1.5) 0;
```

Comment:
NIT: this is not being caught by specificity rules, margin-top remains which means the top row of courses isn't aligned to the top of the sidebar.